python/statsforecast/arima.py

```diff
@@ -950,13 +512,16 @@ def arma_css_op(p, x):
                     raise ValueError("non-stationary seasonal AR part from CSS")
             ncond = 0
         if transform_pars:
-            init = ARIMA_invtrans(init, arma)
-            if arma[1] > 0:
-                ind = arma[0] + np.arange(arma[1])
-                init[ind] = maInvert(init[ind])
-            if arma[3] > 0:
-                ind = np.sum(arma[:3]) + np.arange(arma[3])
-                init[ind] = maInvert(init[ind])
+            candidate = ARIMA_invtrans(init, arma)
+            phi, _ = arima_transpar(candidate, arma, transform_pars)
+            if np.logical_and(phi > -math.pi / 2, phi < math.pi / 2).all():
```

Comment:
The invtrans is one of the functions that had a bug, so with the bug fixed this can sometimes generate invalid init states and we have to check them.

Reply:
nice